testing/test_inference.py

```diff
@@ -91,7 +91,7 @@ def test_inference(film_bool=False):
     model.eval()
 
     metric_fns = [metrics.dice_score,  # from ivadomed/utils.py
-                  metrics.hausdorff_score,
+                  metrics.hausdorff_2D_score,
```

Comment:
3D? or 2D okay here?

Reply:
I was discussing that with @andreanne-lemay in theory 3D should work everywhere since it fallbacks to 2D when it received a not 3D input